frontend/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+import { BrowserRouter as Router, Redirect, Route } from "react-router-dom";
+
+import { Login } from "./components/Login/Login";
+import { MyRooms } from "./components/MyRooms/MyRooms";
+import { Register } from "./components/Register/Register";
+import { RoomList } from "./components/RoomList/RoomList";
+import { WhiteBoard } from "./components/WhiteBoard/WhiteBoard";
+import { Context } from "./context/appContext";
+
+function App() {
+    const [
+        {
+            sessionStorageContext: { username }
+        }
+    ] = useContext(Context);
```

Comment:
I can't see you anywhere using context provider. Moreover you could wrap this `useContext(Context)` in some function `useAppContext()` to avoid doing one additional import.